plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -2133,6 +2134,22 @@ public void testProjectionPushdownMultipleRows()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testPartitionFilterIncluded()
+    {
+        Session session = Session.builder(getSession())
+                .setCatalogSessionProperty(getSession().getCatalog().orElseThrow(), "query_partition_filter_required", "true")
+                .build();
+
+        try (TestTable table = new TestTable((sql) -> getQueryRunner().execute(session, sql), "test_no_partition_filter", "(x VARCHAR, part VARCHAR) WITH (PARTITIONED_BY = ARRAY['part'])")) {
+            assertUpdate(session, "INSERT INTO " + table.getName() + "(x, part) VALUES('a', 'part_a')", 1);
+            assertUpdate(session, "INSERT INTO " + table.getName() + "(x, part) VALUES('b', 'part_b')", 1);
```

Comment:
Can we merge the two insert statement

Reply:
Can we use this query `public TestTable(SqlExecutor sqlExecutor, String namePrefix, String tableDefinition, List<String> rowsToInsert)`  - where we could specify the rows to insert.